.evergreen/config.yml

```diff
@@ -488,6 +488,12 @@ functions:
             export SINGLE_MONGOS_LB_URI="${MONGODB_URI}"
             export MULTI_MONGOS_LB_URI="${MONGODB_URI}"
           fi
+          if [ -n "${TEST_INDEX_MANAGEMENT}" ]; then
```

Comment:
Let's disable xtrace here to avoid printing these secrets to the logs.

Reply:
Done